locale/fr_FR.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -245,7 +245,7 @@ msgstr ""
 # Please mind the whitespace at the end.
 #: index.php:1219 index.php:1377
 msgid "PIN: "
-msgstr ""
+msgstr "PIN:"
```

Comment:
Missing space.

Reply:
Yes, indeed! Thanks for your eagle eyes.

This would a perfect example where you can use the new `Insert a suggestion` feature of GitHub. Do you want to try that out? If yes, just start an answer and then click on the first icon with the +/-.

Reply:
```suggestion
msgstr "PIN: "
```